README.md

```diff
@@ -32,15 +32,19 @@ Policy](https://kubernetes.io/releases/version-skew-policy/).
 | Version - Branch            | Kubernetes branch/version       | Maintenance status |
 | --------------------------- | ------------------------------- | ------------------ |
 | CRI-O HEAD - main           | Kubernetes master branch        | ✓                  |
+| CRI-O 1.27.x - release-1.27 | Kubernetes 1.27 branch, v1.27.x | =                  |
+| CRI-O 1.26.x - release-1.26 | Kubernetes 1.26 branch, v1.26.x | =                  |
+| CRI-O 1.25.x - release-1.25 | Kubernetes 1.25 branch, v1.25.x | =                  |
 | CRI-O 1.24.x - release-1.24 | Kubernetes 1.24 branch, v1.24.x | =                  |
-| CRI-O 1.23.x - release-1.23 | Kubernetes 1.23 branch, v1.23.x | =                  |
-| CRI-O 1.22.x - release-1.22 | Kubernetes 1.22 branch, v1.22.x | =                  |
-| CRI-O 1.21.x - release-1.21 | Kubernetes 1.21 branch, v1.21.x | =                  |
+| CRI-O 1.23.x - release-1.23 | Kubernetes 1.23 branch, v1.23.x | ✗                  |
+| CRI-O 1.22.x - release-1.22 | Kubernetes 1.22 branch, v1.22.x | ✗                  |
+| CRI-O 1.21.x - release-1.21 | Kubernetes 1.21 branch, v1.21.x | ✗                  |
 
 Key:
 
 - `✓` Changes in the main Kubernetes repo about CRI are actively implemented in CRI-O
 - `=` Maintenance is manual, only bugs will be patched.
+- `✗` Following Kubernetes support, these releases are end of life.
```

Comment:
I don't know if this is necessary. do you think it makes the document clearer? it would become an unwieldy list

Reply:
That makes sense, I removed them.